evdns.c

```diff
@@ -863,6 +863,19 @@ reply_schedule_callback(struct request *const req, u32 ttl, u32 err, struct repl
 		&d->deferred);
 }
 
+
+#define _QR_MASK    0x8000
+#define _OP_MASK    0x7800
+#define _AA_MASK    0x0400
+#define _TC_MASK    0x0200
+#define _RD_MASK    0x0100
+#define _RA_MASK    0x0080
+#define _Z_MASK     0x0040
+#define _AD_MASK    0x0020
+#define _CD_MASK    0x0010
+#define _RCODE_MASK 0x000f
+#define _Z_MASK_DEPRECATED 0x0070
```

Comment:
what for?

Reply:
A lot of RFCs reference that this field be set zero, but with the advent of dnssec, they have been used. Verbosity only.